packages/paste-core/components/popover/src/types.ts

```diff
@@ -0,0 +1,13 @@
+import type {BadgeProps} from '@twilio-paste/badge';
+import type {ButtonProps} from '@twilio-paste/button';
+import type {BoxProps} from '@twilio-paste/box';
+
+export type ButtonBadgeProps = Omit<ButtonProps, 'variant' | 'children'> & Pick<BadgeProps, 'children' | 'variant'>;
```

Comment:
Do you need this type? The reason I ask is that where you use it 

`const ButtonBadge = React.forwardRef<HTMLButtonElement, ButtonBadgeProps>`

It feels like you might just be able to use `BadgeProps` directly from Badge instead?